server/api/models/document.dto.ts

```diff
@@ -1,22 +1,22 @@
-import { Copy } from '../services/copyManagement.service';
+import { Document } from '../interfaces/documents';
 
 export class TCDocumentDTO {
```

Comment:
Please change property back to content as 
1) content is more accurate (it's the actual content of the document)
2) this breaks the FE as that depends on the property being named content